src/components/Locations.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import GhibliAPI from "./GhibliAPI";
```

Comment:
Since you're only accessing one thing from `GhibliAPI`, you _could_ do:

`import { getLocations } from './GhibliAPI'`

But whatever style you like!

Reply:
I am getting an import error if I do destructuring.
`Attempted import error: 'getLocations' is not exported from './GhibliAPI'.`